include/libsemigroups/konieczny.hpp

```diff
@@ -385,6 +385,18 @@ namespace libsemigroups {
       _rho_orb.cache_scc_multipliers(true);
     }
 
+    //! Initialise a Konieczny instance to its default state (or an
+    //! indistinguishable one).
+    //!
+    //! \parameters
+    //! (None)
```

Comment:
```suggestion
```
For v3, I think we're opting to not including `\parameters (None)` and `\returns (None)`.